rule/string-format.go

```diff
@@ -125,26 +128,33 @@ func (w lintStringFormatRule) parseArgument(argument any, ruleNum int) (scope st
 		w.configError("regex is too small (regexes should begin and end with '/')", ruleNum, 1)
 	}
 
-	// Parse rule scope
-	scope = stringFormatSubruleScope{}
-	matches := parseStringFormatScope.FindStringSubmatch(rule[0])
-	if matches == nil {
-		// The rule's scope didn't match the parsing regex at all, probably a configuration error
-		w.parseError("unable to parse rule scope", ruleNum, 0)
-	} else if len(matches) != 4 {
-		// The rule's scope matched the parsing regex, but an unexpected number of submatches was returned, probably a bug
-		w.parseError(fmt.Sprintf("unexpected number of submatches when parsing scope: %d, expected 4", len(matches)), ruleNum, 0)
-	}
-	scope.funcName = matches[1]
-	if len(matches[2]) > 0 {
-		var err error
-		scope.argument, err = strconv.Atoi(matches[2])
-		if err != nil {
-			w.parseError("unable to parse argument number in rule scope", ruleNum, 0)
+	// Parse rule scopes
+	rawScopes := strings.Split(rule[0], ",")
```

Comment:
I'm a bit worried about the fact people may use `, `, or ` ,`, or ` , ` as delimiter.

So maybe, you could use strings.TrimSpace on the rawScope in the for loop

Also, xhecking that the rawScope is not empty could avoid surprises and bugs

I'm pretty sure someone would face a 

`foo,,bar`, or `foo , , bar`

No matter the implementation choice to either clean them, or forbid them. It should be covered by a test